.github/CODEOWNERS

```diff
@@ -33,6 +33,7 @@
 # files/folders are created in the following directories, these codeowners would
 # be superseded by the relevant codeowners mentioned elsewhere in this file.
 # (Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)
+* @oppia/owners
```

Comment:
I think this way we won't be able to ensure that all the files are assigned to a specific person.

Assume someone added a new file and forgot to add code owner for that file in that case lint test won't catch it as this line covers all the files and we or owners end up owning a lot of files.

Possible solution:
Do not use lines 36-44 in lint checks for checking whether all the files are owned by someone. wdyt? (I think this is an existing issue and we should fix it separately.)

Reply:
Ah, actually maybe this isn't an issue if we already have a lint check. That tells me a PR can't get merged if its missing code ownership--that seems to solve the same problem, right? In that case, I think I can remove this.

I don't have strong of an opinion at the moment on whether the other blank ownership lines should be removed. I think that has more to do with our general policy around how to organize codeowners & add them over time. I haven't spent much time thinking about that on the web side yet.